api.php

```diff
@@ -86,8 +87,44 @@
 		->withStatus(200);
 });
 
+//finds User by Email TODO modify route
```

Comment:
Why are we modifying the route and in what way?

Reply:
Didn't you do this route? I don't know what this comment is referring to. Or are you asking for me to learn why?

Reply:
idk... theres a note in there to modify the route and I dont know why... I may have written it but I dont remember (bad comments are bad because of things like this)